lib/Doctrine/Common/DataFixtures/Loader.php

```diff
@@ -79,14 +77,14 @@ class Loader
      *
      * @return array $fixtures Array of loaded fixture object instances.
      */
-    public function loadFromDirectory($dir)
+    public function loadFromDirectory(string $dir): array
```

Comment:
This is a BC break for extending classes and as such, should be contributed to 2.x
I can create that branch if you feel up to that.

Reply:
I can rollback methods type hinting to not create breaking changes

Reply:
If you want this merged on 2.x, you definitely will have to.